scripts/ci/test:downstream

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -e # exit on error
+set -x # echo commands
+
+pushd `python -c "import site; print(site.getsitepackages()[0])"`
+py.test distributed/bokeh
```

Comment:
Given `-e` flag, this will fail this job when test fail (or am I missing something?).

Reply:
Ah, yes apparently so. I thought it meant something slightly different.